eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/LiteConsumer.java

```diff
@@ -126,10 +127,13 @@ public boolean subscribe(List<String> topicList, String url) throws Exception {
         long startTime = System.currentTimeMillis();
         String target = selectEventMesh();
         String subRes = "";
+        CloseableHttpClient httpClient = setHttpClient();
```

Comment:
Can we use `try with resource` here?
```java
try (CloseableHttpClient httpClient = setHttpClient()){

}catch(){

}
```
And I think it's not very good to do a conversion on `Exception` here, which will hide the source of the `Exception`. If the `Exception` is not currently handled, the best way may to throw it directly instead of wrapping it.

Reply:
Thanks for your suggestion. I changed to use `try with resource`.